README.md

```diff
@@ -191,21 +208,9 @@ kubectl get claim
 kubectl get composite
 kubectl get managed
 ```
+>_Note: It may take some time for all managed resources to create successfully. Please reconcile using the AWS console._
```

Comment:
What is the intent of the messaging to "Please reconcile using the AWS console"? Ae we trying to say something like "You can also check the progress of these resources in the AWS console"?

Reply:
Yes, that was the original intent. It makes sense just to remove it.